101-cognitive-services-universalkey/azuredeploy.json

```diff
@@ -1,42 +1,42 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "cognitiveServiceName": {
-            "type": "string",
-            "defaultValue": "[concat('CognitiveService-', uniqueString(resourceGroup().id))]",
-            "metadata":{
-                "description": "That name is the name of our application. It has to be unique.Type a name followed by your resource group name. (<name>-<resourceGroupName>)"
-            }
-        },
-        "location": {
-            "type": "string",  
-            "defaultValue": "[resourceGroup().location]",  
-            "metadata": {  
-                    "description": "Location for all resources."  
-            }  
-        },
-        "sku":{
-            "type": "string",
-            "defaultValue": "S0",
-            "allowedValues": [
-                "S0"
-            ]
-        }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "cognitiveServiceName": {
+      "type": "string",
+      "defaultValue": "[concat('CognitiveService-', uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "That name is the name of our application. It has to be unique.Type a name followed by your resource group name. (<name>-<resourceGroupName>)"
+      }
     },
-    "resources": [
-        {
-            "apiVersion": "2017-04-18",
-            "name": "[parameters('cognitiveServiceName')]",
-            "location": "[parameters('location')]",
-            "type": "Microsoft.CognitiveServices/accounts",
-            "kind": "CognitiveServices",
-            "sku": {
-                "name": "[parameters('sku')]"
-            },
-            "properties": {
-                "statisticsEnabled": false
-            }
-        }
-    ]
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "sku": {
+      "type": "string",
+      "defaultValue": "S0",
+      "allowedValues": [
+        "S0"
+      ]
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.CognitiveServices/accounts",
+      "apiVersion": "2017-04-18",
+      "name": "[parameters('cognitiveServiceName')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "[parameters('sku')]"
+      },
+      "kind": "CognitiveServices",
+      "properties": {
+        "statisticsEnabled": false
+      }
+    }
+  ]
 }
```

Comment:
@bmoore-msft should a blank line be added to fix the `no newline at end of file` message?

Reply:
fixed.